arches_her/datatypes/bngcentrepoint.py

```diff
@@ -96,16 +96,17 @@ def validate(self, value, row_number=None, source=None, node=None, nodeid=None,
             "TV",
             "TW",
         ]
-        try:
-            # CS - Validation for datatype.  Replicates functionality in widget which will be removed once datatype validation is fixed.
-            firstTwoCharacters = value[0:2]
-            numberElement = value[2:]
+        if value is not None:

```

Comment:
@SDScandrettKint Could you put a comment in the code to say this is a temporary fix until a change is made in core Arches?